modules/conflict_resolver/php/NDB_Menu_Filter_Form_conflict_resolver.class.inc

```diff
@@ -389,75 +389,7 @@ class NDB_Menu_Filter_Form_Conflict_Resolver extends NDB_Menu_Filter_Form
 
         return true;
     }
-    /**
-     * Returns a list of candidates, users, etc.
-     * Need override for when Examiner field is conflicting,
-     * need Examiner's full name
-     *
-     * @return array
-     * @access private
-     */
-    function _getFullList()
-    {
-        // create DB object
-        $DB =& Database::singleton();
-
-        $qparams = array();
-        // add the base query
-        $query  = '';
-        $query .= $this->_getBaseQuery();
-
-        $filterdetails = $this->_getBaseFilter();
-        $query        .= $filterdetails['clause'];
-        $qparams       = $filterdetails['params'];
-        // apply ORDER BY filters
-        $query .= " ORDER BY ";
-        if (!empty($this->filter['order'])) {
-            $query .= $this->filter['order']['field']
-                ." ".$this->filter['order']['fieldOrder'].", ";
-        }
-        $query .= $this->order_by;
-        // get the list
-
-        $pageNum = 1;
```

Comment:
@kongtiaowang i discovered the issue
in this file, remove lines 422-425, 431-432, and alter line 433 to remove the `$limit` append

then do the same in the other file